core/stream/convert.go

```diff
@@ -396,7 +398,7 @@ func writeUintClamped(bs *binary.BitStream, bits uint64, count uint32) {
 	bs.Write(u64.Min(bits, limit), count)
 }
 
-func ftouMinMax(count int, dst, src buf, min, max *float64) error {
+func readMinMax(count int, src buf, min, max *float64) error {
```

Comment:
why the rename? I don't see how the new name is better than the old.

Reply:
I guess because he got rid of dst?

Reply:
remove the dst as it is not used.